devdocs/settingUp.md

```diff
@@ -30,19 +30,18 @@ Important: When a version is specified, please install that version instead of t
    Instructions are at https://developers.google.com/eclipse/docs/install-eclipse-4.4 <br>
    Note: Sometimes the update site for the GAE plug-in does not work. In which case, 
    follow the instructions at https://developers.google.com/eclipse/docs/install-from-zip.
-5. Install Google App Engine SDK (version specified by Gradle). <br>
-   Run the command `./gradlew appengineDownloadSdk`.<br>
-   Go to `Window → Preferences → Google → App Engine` (Mac: `Eclipse → Preferences → Google → App Engine`), click the `Add` button,
-   and point it to where Gradle kept the SDK. This directory can be found by running the command `./gradlew printUserHomeDir`.<br>
-   Further instructions for installing can be found at https://developers.google.com/eclipse/docs/using_sdks.
-6. Install the latest [TestNG Eclipse plugin](http://testng.org/doc/download.html).
+5. Install the latest [TestNG Eclipse plugin](http://testng.org/doc/download.html).
 
 ## Setting up the dev server
 `Dev server` means running the server in your own computer.
 
 1. Fork our repo at https://github.com/TEAMMATES/repo. Clone that fork to your hard disk.
-2. Configure Eclipse (if you worry that these settings will interfere with your 
-    other projects, you can use a separate eclipse instance for TEAMMATES):
+2. Run the command `./gradlew appengineDownloadSdk` to obtain the Google App Engine SDK (version specified by Gradle).
```

Comment:
Remove `(version specified by Gradle)`? 
or change to `(version specified in abc.gradle)`